src/main/java/uk/gov/hmcts/reform/ccd/document/am/configuration/AuthCheckerConfiguration.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.hmcts.reform.ccd.document.am.configuration;
+
+import com.google.common.collect.ImmutableSet;
+
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLContext;
+import javax.servlet.http.HttpServletRequest;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.apache.http.client.HttpClient;
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.conn.ssl.TrustStrategy;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.ssl.SSLContexts;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "security")
+@Slf4j
+public class AuthCheckerConfiguration {
+
+    List<String> authorisedServices;
+    List<String> authorisedRoles = new ArrayList<>();
+
+    public List<String> getAuthorisedServices() {
+        return authorisedServices;
+    }
+
+    public void setAuthorisedServices(List<String> authorisedServices) {
+        this.authorisedServices = authorisedServices;
+    }
+
+    public List<String> getAuthorisedRoles() {
+        return authorisedRoles;
+    }
+
+    @Bean
+    public Function<HttpServletRequest, Collection<String>> authorizedServicesExtractor() {
+        //log.info(String.format("Configured authorised services: %s", String.join(", ", authorisedServices)));
```

Comment:
can we change the logging level to debug and enable whatever necessary for debugging purpose.

Reply:
can you please update this element?